RELEASE.md

```diff
@@ -76,6 +76,8 @@ Tag the release
 * click pre-release for beta releases
 * publish the release
 
+[Update ember-new-output and ember-addon-output.](https://gist.github.com/nathanhammond/e0a55b4d0328b45b8ef5)
```

Comment:
Should probably also call out the additions to the changelog:
[`ember new` diff](https://github.com/ember-cli/ember-new-output/compare/v2.6.0-beta.2...v2.6.0-beta.3)
[`ember addon` diff](https://github.com/ember-cli/ember-addon-output/compare/v2.6.0-beta.2...v2.6.0-beta.3)